### PR TITLE
Add more detail about container images and referencing them by digest

### PR DESCRIPTION
**Pull request by KhushPatibandha**
Explained image digest by making a new topic and added links to help understand image digest better.
issue #39870

### CODE REVIEW

**Comment by sftim** [comment]
/retitle Add more detail about container images and referencing them by digest

File: content/en/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -44,6 +44,28 @@ There are additional rules about where you can place the separator
...
+
+Using digest can provide a more secure and reliable way of referencing 
+container images because image digest is based on the contents of the image, 
+it provides a way to ensure that the image has not been tampered or modified with.
+
+When referencing an image in a pod, you can use either an image tag or an image digest. 
+However, using an image digest can be more reliable that image tag since it ensures that 
+the exact version of the image is used every time the pad is created or updated.
+
+For more information refer [Using container image digests](https://cloud.google.com/kubernetes-engine/docs/archive/using-container-images).
```
Comment (sftim) [inline content/en/docs/concepts/containers/images.md:67]: This isn't quite the right thing. I'd omit this line until we have our own page to link to.
```suggestion
```

**Review by sftim — COMMENTED** [review]
One (important) comment

**Comment by KhushPatibandha** [comment]
Hey, @sftim hope you doing great. What's the update on this PR?

File: content/en/docs/concepts/containers/images.md [hunk 2]
```diff
@@ -44,6 +44,26 @@ There are additional rules about where you can place the separator
...
+unique identifier for a specific version of a container image. 
+It is calculated using a hash function that takes account of the contents of all the layers in the image.
+
+The image digest is a fixed identifier that can't be changed, even if the image is updated. 
+
+An image digest is a long string of characters, typically in 
+the form of a [SHA-256](https://github.com/opencontainers/image-spec/blob/main/descriptor.md#sha-256) 
+hash, this uniquely identifies a specific version of a container image.
+
+Using digest can provide a more secure and reliable way of referencing
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/containers/images.md:59]: I think this particular reiterates the point made in the second sentence and could be made idiomatic. Please could you rephrase it to ensure that the points being made aren't redundant?

File: content/en/docs/concepts/containers/images.md [hunk 3]
```diff
@@ -44,6 +44,26 @@ There are additional rules about where you can place the separator
...
+the form of a [SHA-256](https://github.com/opencontainers/image-spec/blob/main/descriptor.md#sha-256) 
+hash, this uniquely identifies a specific version of a container image.
+
+Using digest can provide a more secure and reliable way of referencing 
+container images because image digest is based on the contents of the image, 
+it provides a way to ensure that the image has not been tampered or modified with.
+
+When referencing an image in a pod, you can use either an image tag or an image digest. 
+However, using an image digest can be more reliable that image tag since it ensures that 
+the exact version of the image is used every time the pad is created or updated.
```
Comment (Ritikaa96) [inline content/en/docs/concepts/containers/images.md:65]:
```suggestion
the exact version of the image is used every time the pod is created or updated.
```

File: content/en/docs/concepts/containers/images.md [hunk 4]
```diff
@@ -44,6 +44,26 @@ There are additional rules about where you can place the separator
...
+An image digest is a long string of characters, typically in 
+the form of a [SHA-256](https://github.com/opencontainers/image-spec/blob/main/descriptor.md#sha-256) 
+hash, this uniquely identifies a specific version of a container image.
+
+Using digest can provide a more secure and reliable way of referencing 
+container images because image digest is based on the contents of the image, 
+it provides a way to ensure that the image has not been tampered or modified with.
+
+When referencing an image in a pod, you can use either an image tag or an image digest. 
+However, using an image digest can be more reliable that image tag since it ensures that
```
Comment (Ritikaa96) [inline content/en/docs/concepts/containers/images.md:64]:
```suggestion
However, using an image digest can be more reliable than image tag since it ensures that 
```

Comment (Ritikaa96) [reply content/en/docs/concepts/containers/images.md:64]: small nits

**Review by Ritikaa96 — CHANGES_REQUESTED** [review]

File: content/en/docs/concepts/containers/images.md [hunk 5]
```diff
@@ -44,6 +44,26 @@ There are additional rules about where you can place the separator
...
+
+The image digest is a fixed identifier that can't be changed, even if the image is updated. 
+
+An image digest is a long string of characters, typically in 
+the form of a [SHA-256](https://github.com/opencontainers/image-spec/blob/main/descriptor.md#sha-256) 
+hash, this uniquely identifies a specific version of a container image.
+
+This is more secure and reliable way of referencing 
+container images because image digest is based on the contents of the image, 
+it provides a way to ensure that the image has not been tampered or modified with.
```
Comment (Ritikaa96) [inline content/en/docs/concepts/containers/images.md:61]:
```suggestion
it provides a way to ensure that the image has not been tampered with or modified.
```
'Modified with' doesn't sound right. what it is modified with? did you mean tampered with?

File: content/en/docs/concepts/containers/images.md [hunk 6]
```diff
@@ -44,6 +44,26 @@ There are additional rules about where you can place the separator
...
+unique identifier for a specific version of a container image. 
+It is calculated using a hash function that takes account of the contents of all the layers in the image.
+
+The image digest is a fixed identifier that can't be changed, even if the image is updated. 
+
+An image digest is a long string of characters, typically in 
+the form of a [SHA-256](https://github.com/opencontainers/image-spec/blob/main/descriptor.md#sha-256) 
+hash, this uniquely identifies a specific version of a container image.
+
+This is more secure and reliable way of referencing
```
Comment (Ritikaa96) [inline content/en/docs/concepts/containers/images.md:59]:
```suggestion
This is a more secure and reliable way of referencing 
```

**Review by Ritikaa96 — CHANGES_REQUESTED** [review]

**Comment by Ritikaa96** [comment]
Few more nits @KhushPatibandha

**Comment by sftim** [comment]
@KhushPatibandha the feedback in https://github.com/kubernetes/website/pull/40402#issuecomment-1827213360 looks worth fixing - I'd expect a bug report if we left it in. Could your revise again?

Ideally, also squash your commits.

**Comment by divya-mohan0209** [comment]
Hello @KhushPatibandha , despite reminders since there have been no commits/comments from your side, we are closing this PR. 

Please note that when you have the bandwidth, you may choose to resume work by issuing the /reopen command on this PR.

/close
